tmpl/page.tmpl

```diff
@@ -2,12 +2,14 @@
 <html lang="en">
     <head>
         <title>{{title}} | PlayCanvas API Reference</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
```

Comment:
Lighthouse complains about this:
https://web.dev/meta-viewport/
Mobile Safari actually ignores this setting. But yeah, I would recommend deleting it as it seems to be considered 'bad practice'.
If you haven't used Lighthouse before (in Chrome Dev Tools), check it out. It's awesome and I've used it to tune all the marketing pages in playcanvas.com